src/sentry/tasks/unmerge.py

```diff
@@ -342,7 +344,7 @@ def repair_tag_data(caches, project, events):
             env_name,
         )
         for key, values in keys.items():
-            tagstore.get_or_create_group_tag_key(
+            _, created = tagstore.get_or_create_group_tag_key(
```

Comment:
This assignment isn't necessary/is shadowed below on L358.

Reply:
Oops ya was part of debugging, will remove.